public/index.html

```diff
@@ -0,0 +1,149 @@
+<!doctype html>
+<html>
+<head>
+
+	<title>Ofek Chat</title>
+
+
+</head>
+
+
+<script src="/socket.io/socket.io.js"></script>
+
+<link rel="stylesheet" href="/css/bootstrap.min.css">
+<link rel="stylesheet" href="/css/animate.css">
+<link rel="stylesheet" type="text/css" href="/css/index.css">
+<script src="/js/jquery.min.js"></script>
+<script src="/js/jquery-extends.js"></script>
+<script src="/js/bootstrap.min.js"></script>
+
+
+<script src="/js/UI.js"></script>
+<script src="/js/User.js"></script>
+<script src="/js/index.js"></script>
+
+
+<body>
+<ul id="messages"></ul>
+
+<div class="panel panel-default" id = "commander">
+	<div class="panel-body">
+		<form action="">
+			<div class="form-group">
+				<label for="name">Name:</label>
+				<input type="name" class="form-control" id="name">
+			</div>
+			<input id="m" autocomplete="off" /><button class="btn btn-success">Send</button>
+			<a data-toggle="modal" data-target="#myModal">Show Commands</a>
+		</form>
+	</div>
+</div>
+
+
+
+<!-- Modal -->
+<div id="myModal" class="modal fade" role="dialog">
+	<div class="modal-dialog">
+
+		<!-- Modal content-->
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal">&times;</button>
+				<h4 class="modal-title">Commands</h4>
+			</div>
+			<div class="modal-body">
+				<p><b>/color</b> 'red' or 'ff0000' - will color the text red</p>
```

Comment:
Instead of writing all in the view, you should move it to constants (and recommended to other file) and add it on document ready event:

```JS
const animationArray = ['bounce','flash','pulse','rubberBand'...]
    $(document).ready(()=> {
        for (let animation of animationArray) {
            $('.modal-body').append(`<div>${animation}</div>`);
        }
        $('.modal-body').append(`<div>will animate the text</div>`);
    });
```